rpm/iptables.spec

```diff
@@ -1,6 +1,6 @@
 Name:       iptables
 Summary:    Tools for managing Linux kernel packet filtering capabilities
-Version:    1.8.2
+Version:    1.8.7
 Release:    1
 Group:      System/Base
```

Comment:
You could drop all Group tags. They are [deprecated](https://sailfishos.org/wiki/Software_Packaging#RPM_packaging_Guidelines).

Reply:
Right, cleaned up that and force pushed as minor change.